Makefile

```diff
@@ -588,6 +588,11 @@ RUSTC		:= rustc --target=$(CONFIG_LLVM_TARGET_ARCH)
 else
 RUSTC		:= rustc
 endif
+ifneq ("$(origin LLVM_TARGET_ARCH)","undefined")
+CARGO		:= cargo --target=$(CONFIG_LLVM_TARGET_ARCH)
```

Comment:
Hum 🤔  Where is this used? ...and for which reasons do we need cargo in the core repo?

Reply:
The user should be able to set his cargo path this way. Something like `make CARGO=xcargo or /path/to/cargo`.

Reply:
I mean, I do not see a build rule using `$(CARGO)`. The thing is, if cargo is used by a external support lib, you should implement this with this library's `Makefile.rules`. We could also add support to load external `Makefile.build`. @marcrittinghaus looked into this.

Reply:
I've moved it to the library's `Makefile.rules`[1] :confetti_ball: .

[1] https://github.com/unikraft/lib-rust/blob/staging/Makefile.rules#L39